libnavigation-core/src/main/java/com/mapbox/navigation/core/internal/InternalEx.kt

```diff
@@ -0,0 +1,27 @@
+package com.mapbox.navigation.core.internal
+
+import com.mapbox.navigation.core.trip.session.MapboxTripSession
+
```

Comment:
we have pattern `**Ex` for extensions, but it's more like utils. Could we rename file to `InternalUtils` or smth like this?

Reply:
done :+1: